mlflow/protos/databricks_artifacts.proto

```diff
@@ -0,0 +1,129 @@
+syntax = "proto2";
```

Comment:
@arjundc-db Can we update this proto file to include the `page_token` for the ListArtifacts API?

Reply:
Nevermind!